authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationPlugin.java

```diff
@@ -786,7 +783,9 @@ private void createRangerServiceIfNecessary(Map<String, String> config, String s
     try {
       rangerClient.getService(serviceName);
     } catch (RangerServiceException rse) {
-      if (ClientResponse.Status.NOT_FOUND.equals(rse.getStatus())) {
+      if (Boolean.parseBoolean(
+              config.get(RangerAuthorizationProperties.RANGER_SERVICE_CREATE_IF_ABSENT))
+          && ClientResponse.Status.NOT_FOUND.equals(rse.getStatus())) {
```

Comment:
You have changed the original design:
Before:  If `RANGER_SERVICE_CREATE_IF_ABSENT` is false, there will be no exception in any condition. 
Now:  If `RANGER_SERVICE_CREATE_IF_ABSENT` is false and the service is unavailable, you will throw 

```java
throw new AuthorizationPluginException(
            "Fail to get ranger service name %s, exception: %s", serviceName, rse.getMessage());
```

Reply:
Yes, the change seems more reasonable for me.

Reply:
@xunliu, could you please help confirm the original design?

Reply:
I think it ok